api/src/wsmanager.py

```diff
@@ -86,7 +89,7 @@ async def consume(
         try:
             message = json.loads(json_message)
         except json.JSONDecodeError as e:
-            print(e)
+            logger.info("invalid json received from client", exc_info=e)
```

Comment:
These should be single quotes to be consistent.

Reply:
good catch, we should have a formatter check this ;)